RFC/0001-lightweight-rfc-process.md

```diff
@@ -0,0 +1,117 @@
+# RFC 0001: Lightweight RFC Process for IPFS Specifications
+
+- Start Date: 2022-06-10
+- Related Issues:
+  - [ipfs/specs/issues/286](https://github.com/ipfs/specs/issues/286)
+
+# Summary
+
+This Request for Change (RFC) introduces lightweight RFC process
+for IPFS specifications [repository][1].
+
+[1]: https://github.com/ipfs/specs/
+
+# Motivation
+
+Protocol design discussions often take place in a repository of an IPFS
+implementation. Often lacking useful input from IPFS Stewards, other
+implementations, service operators and the wider IPFS Community.
+
+IPFS Project needs a mechanism for proposing and evaluating specification
+improvements that are not tied to a specific programming language
+or implementation of IPFS.
+
+# Detailed design
+
+Adopt an informal RFC process for [ipfs/specs][1] repository, providing
+minimal structure for opening, reviewing, and merging specification changes.
+
+## Opening
+
+Changes to IPFS specifications can be proposed by opening a PR against the
+repository, and including a short "request for change" document based on the
+template in `RFC/0000-template.md`.
+
+## Reviewing
+
+IPFS Stewards will review new RFC PRs during weekly triage.
+
+IPFS Community is encouraged to participate in the review process.
+
+Final decision belongs to IPFS Stewards.
+
+## Merging
+
+RFC can be merged only after two Stewards approve it.
+
+RFC number is assigned before the merge.
+
+RFC author and two approving Stewards are added to CODEOWNERS file to be
+automatically asked to review any future changes to files added or modified
+by the RFC.
+
+## Long-term plan
+
+IPFS Stewards will adjust the process based on usage patterns.
+
+## Design rationale
+
+We want to empower IPFS community and implementers with the ability to propose
+changes in a well-known way, without introducing much overhead.
+
+Guiding principles:
+- No new tooling
+  - Reuse Markdown, Git, and existing PR review process
+- Convention over Byzantine process
+  - Proposing a new RFC should have low cognitive overhead, allowing us to
+    focus on specs
+  - Reuse existing Git developer accounts and reputation attached to them
+  - One should be able to create a valid RFC without reading a long explainer
+    like this one. Looking at past RFCs, and then copying a template and
+    opening a PR with their proposal should be more than enough.
+
+### User benefit
+
+End users will indirectly benefit from a healthy RFC process being in place:
+
+- IPFS community members will be able to use RFC drafts for evaluating ideas
+  before investing time into building new things.
+- The bar for creating a brand new IPFS implementation will be lowered, and
+  existing implementations will be able to propose improvements for others to
+  adopt. This removes the soft vendor lock-in present when the oldest
+  implementation is considered as the reference standard and source of truth.
+- IPFS Implementers will have a better understanding why certain design
```

Comment:
```suggestion
- IPFS implementers will have a better understanding of why certain design
```
lowercase the "i" here since it's not a formal group name